### PR TITLE
feat: added pomodoro timer

### PR DESCRIPTION
**Pull request by krate27**

### CODE REVIEW

File: src/Components/Cards/taskCard.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import "../Cards/taskCard.css";
 import { colors } from "./cardColor";
 import { useTask } from "../../Context/taskContext";
 import { DeleteTodo } from "../helperFunctions/deleteTodo";
+import { Link } from "react-router-dom";
 const TaskCard = ({ items, setTaskDetails, setShowModal, showModal }) => {
```
Comment (hinanshisuthar) [inline src/Components/Cards/taskCard.jsx:6]: Good use of destructuring made here

File: src/Components/Pomodoro/pomodoro.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { buildStyles, CircularProgressbar } from "react-circular-progressbar";
+import "react-circular-progressbar/dist/styles.css";
+import "../Pomodoro/pomodoro.css";
+
+const Pomodoro = () => {
+  const location = useLocation();
+  const { items } = location.state;
+  const [timeId, setTimeId] = useState(0);
+  const [continuous, setContinuous] = useState(0);
+  if (continuous > items.time * 60) resetTimerBtn(timeId, setContinuous);
```
Comment (hinanshisuthar) [inline src/Components/Pomodoro/pomodoro.jsx:19]: Everything used is in the form of components. Really good job done

**Review by hinanshisuthar — APPROVED** [review]
Liked how to broke long tasks into simple/short ones. Making components and reusing them later everywhere. Good to merge.

File: src/Components/Todos/todos.jsx [hunk 3]
```diff
@@ -3,55 +3,68 @@ import { useTask } from "../../Context/taskContext";
...
+                setTaskDetails={setTaskDetails}
+                setShowModal={setShowModal}
+                showModal={showModal}
+              />
+            ))}
+          </div>
+        </>
+      ) : (
+        <>
+          <h2 style={{ textAlign: "center", margin: "1rem" }}>
```
Comment (kavyakunder) [inline src/Components/Todos/todos.jsx:58]: The styling for h2 can be shifted to your todos.css file.

**Review by kavyakunder — APPROVED** [review]
Just a minor suggestion I have given.
The code is well written which makes it easily understandable.

**Pull request merged**